Firmware/UI/Screens/MainPage.cpp

```diff
@@ -60,20 +60,20 @@ void MainPage::DigitalGainButtonHandler(void* sender)
 
 void MainPage::Draw(IPainter* painter)
 {
-    /*painter->DrawIcon(apertus_text.pixel_data_text, 58, 89, apertus_text.width, apertus_text.height,
+    painter->DrawIcon(apertus_text.pixel_data_text, 58, 89, apertus_text.width, apertus_text.height,
```

Comment:
What is pixel_data_text?

Reply:
It was the instance for the old header file

Reply:
As you are doing this PR for new approach, please adjust it accordingly.

Reply:
Okay!